component/azstorage/azauthmsi.go

```diff
@@ -52,6 +53,11 @@ type azAuthMSI struct {
 	azAuthBase
 }
 
+func getNextExpiryTimer(token *adal.Token) time.Duration {
+	delay := time.Duration(5+rand.Intn(120)) * time.Second
```

Comment:
I'm very intrigued on why we want an inconsistent time duration interval.

Reply:
Yeah there was in issue with blobfuse2 where certain customers had many concurrent mounts and the time delays were all the same. So requests for authorization would be sent at the same time, and some requests would be denied because they had too many requests at the same time. So this prevents all of those requests from being sent at the same time.